scipy/special/wright_bessel.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import division, print_function, absolute_import
+
+from ._ufuncs import _wright_bessel
+
+
+def wright_bessel(a, b, x):
```

Comment:
Looks like you don't need this Python wrapper here; you can export the ufunc directly (which is preferred-it will mean it is also exported in `cython_special`).

Reply:
This means, the function is accessible as `from scipy.special._ufuncs import _wright_bessel`.

Reply:
Yes, or if you make the ufunc `wright_bessel` then it will get put in the `__all__` for `_ufuncs`, which will make it available as `special.wright_bessel`.

Reply:
Ok. Now it is exposed as `scipy.special.wright_bessel` without leading underline.

Reply:
Sorry to bother you again, but the import does not work anymore after deleting `scipy/special/wright_bessel.py` an deleting the "_" prefix.
Now, there is only the `wright_bessel.pxd` file, the entry in `functions.json` and in `add_newdocs.py`. I get the following error:
```python
from scipy.special import wright_bessel                                 
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-645a9cc16623> in <module>
----> 1 from scipy.special import wright_bessel

~/github/scipy/scipy/special/__init__.py in <module>
    631 from .sf_error import SpecialFunctionWarning, SpecialFunctionError
    632 
--> 633 from . import _ufuncs
    634 from ._ufuncs import *
    635 

_ufuncs.pyx in init scipy.special._ufuncs()

ModuleNotFoundError: No module named 'scipy.special.wright_bessel'
```
What am I missing here?

Reply:
I got it. It works now.